PackagingProjects/CSharpPluginPayload/RepackageCSharpPlugin.ps1

```diff
@@ -1,214 +1,269 @@
-param([string]$buildConfiguration="Debug")
-
-# import GAC assembly
-Add-Type -AssemblyName "System.IO.Compression.FileSystem"
```

Comment:
Unfortunately the diffing algorithm of github is very poor. The files are identical ~70% ... I'll point out the main difference